pkgs/applications/file-managers/gnome-commander/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv
+, lib
+, fetchurl
+, flex
+, itstool
+, pkg-config
+, libxslt
+, wrapGAppsHook
+, exiv2
+, glib
+, gtk2
+, gvfs
+, libgsf
+, libunique
+, poppler
+, taglib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-commander";
+  version = "1.14.3";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/${pname}/${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "sha256-eNjc5w+5IrKQnPdneDBTsIESE6TWpJs4dVEM86hO/Xs=";
+  };
+
+  nativeBuildInputs = [
+    flex
+    itstool
+    pkg-config
+    libxslt
+    (wrapGAppsHook.override { gtk3 = gtk2; })
+  ];
+
+  buildInputs = [
+    exiv2
+    glib
+    gtk2
+    libgsf
+    libunique
+    poppler
+    taglib
+  ];
+
+  patches = [ ./gsettings.patch ];
```

Comment:
This is not actually necessary – the `global_schema_source` will pick it up from the wrapper.